modules/KalturaSupport/components/dualScreen/dualScreen.css

```diff
@@ -155,7 +155,7 @@
     visibility: visible;
     opacity: 1;
 }
-.componentOff{
+.componentOff *{
```

Comment:
@MaksymKorniiov 
I don't like using * - it is too risky. Any chance to be more protective of the entire CSS scope?